src/views/components/timeline/Timeline.web.js

```diff
@@ -0,0 +1,35 @@
+import React, { Fragment } from 'react';
+import { shape, string, number } from 'prop-types';
+import * as ReactTimeline from 'react-timeline-scribble';
+
+const   Timeline  = ( props ) => {
+  const renderEvents = () => {
+    const { data } = props;
+
+    return data.map( dd => (
```

Comment:
`data` is listed as an object in the PropTypes and in the `TIMELINE.md` documentation file you created. However here, we're calling `.map()` on `data` - which will fail immediately since `data` is an object and not an `array`. Can you please revise this? Thanks